mautrix_signal/signal.py

```diff
@@ -149,12 +149,17 @@ async def handle_receipt(sender: 'pu.Puppet', receipt: Receipt) -> None:
 
     async def start(self) -> None:
         await self.connect()
+        self.log.info("Subscribing to users")
         known_usernames = set()
         async for user in u.User.all_logged_in():
             # TODO report errors to user?
-            known_usernames.add(user.username)
-            if await self.subscribe(user.username):
-                asyncio.create_task(user.sync())
+            try:
+                known_usernames.add(user.username)
+                if await self.subscribe(user.username):
+                    asyncio.create_task(user.sync())
+            except Exception as ex:
+                # Don't hold up the whole bridge over this.
+                self.log.warn(f"Failed to subscribe to {user.username}: {ex}")
```

Comment:
https://docs.python.org/3/library/logging.html#logging.Logger.warning

```suggestion
                self.log.warning(f"Failed to subscribe to {user.username}: {ex}")
```